workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -87,14 +93,22 @@ export const WorkflowInstancePageContent: React.FC<{
   );
 
   const workflowdata = assessedInstance.instance?.workflowdata;
-  let instanceVariables;
+  let inputs;
   if (workflowdata) {
-    instanceVariables = {
-      /* Since we are about to remove just the top-level property, shallow copy of the object is sufficient */
+    inputs = {
       ...workflowdata,
     };
-    if (instanceVariables.hasOwnProperty('result')) {
-      delete instanceVariables.result;
+
+    const { schema } = location.state || {};
+
+    // Ensure schema properties are an object and iterate over them
+    const properties = schema.properties || {};
+
+    for (const key of Object.keys(inputs)) {
```

Comment:
I suggest to put this in a separate function getInputsFromVariables that gets the schema and the variables